src/handlers.js

```diff
@@ -1506,6 +1506,22 @@ const handlers = {
 		state.commands.push(dedent, newline, '}', newline);
 	},
 
+	TSModuleBlock(node, state) {
+		state.commands.push(' {', indent, newline);
+		sequence(node.body, state, false, handle);
+		state.commands.push(dedent, newline, '}');
+	},
+
+	TSModuleDeclaration(node, state) {
+		if (node.declare) state.commands.push('declare ');
+		else state.commands.push('namespace ');
```

Comment:
Not sure if this is a correct assumption, but I did not find any other way to extract that `namspace` from the AST